entity-service/src/test/kotlin/com/egm/stellio/entity/config/TestContainersConfiguration.kt

```diff
@@ -0,0 +1,30 @@
+package com.egm.stellio.entity.config
+
+import com.egm.stellio.shared.config.TestContainers
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+
+@TestConfiguration
+class TestContainersConfiguration {
+
+    private val DB_USER = "neo4j"
+    private val DB_PASSWORD = "neo4j_password"
+
+    object EntityServiceTestContainers : TestContainers("neo4j", 7687) {
```

Comment:
Shouldn't the name be something like Neo4JTestContainers ?

Reply:
Because it uses the docker-compose for entity-service but we are just exposing the neo4j service (what we need for now).
maybe, later, we will need to expose other services.
That's why i see it as EntityServiceTestContainers rather than Neo4JTestContainers